### ISSUE TITLE
Автоматический запуск тестов в linux

### ISSUE THREAD

**Issue by Veres72** [xDrivenDevelopment/xUnitFor1C#677]
### Предварительно

* [да ] Вы используете [последнюю версию продукта](https://github.com/xDrivenDevelopment/xUnitFor1C/tree/develop)  ?
* [да ] Вы проверили [Известные проблемы при тестировании](https://github.com/xDrivenDevelopment/xUnitFor1C/wiki/%D0%98%D0%B7%D0%B2%D0%B5%D1%81%D1%82%D0%BD%D1%8B%D0%B5-%D0%BF%D1%80%D0%BE%D0%B1%D0%BB%D0%B5%D0%BC%D1%8B-%D0%BF%D1%80%D0%B8-%D1%82%D0%B5%D1%81%D1%82%D0%B8%D1%80%D0%BE%D0%B2%D0%B0%D0%BD%D0%B8%D0%B8) ?

## Описание

Не молучается сделать автоматический запуск тестов из командной строки linux

**Шаги для воспроизведения**

1. Беру любую конфу ( чистая база, БСП демо, 1с медицина). 
2. Запускаю тесты руками - через "Загрузить тесты"
3. Указываю директорию с тестами - все ок. 
4. Запускаю через командную строку (/opt/1C/v8.3/x86_64/1cv8 1cv8 ENTERPRISE /IBConnectionString "Srvr=1c-dev;Ref=popovei;" /RunModeManagedApplication /Execute "xddTestRunner.epf" /C "xddRun ЗагрузчикКаталога <путь к каталогу тестов> xddReport  ГенераторОтчетаJUnitXML /tmp/report.xml;")

**Ожидаемое поведение:** Автоматический запуск тестов

**Текущее поведение:** Тесты не запускаются

**Укажите:**
* версия продукта 4.0.0.7
* версия платформы 1С 8.3.9.2033
* запуск в пакетном режиме, толстый клиент

Запуск рукаи: 
![ruki](https://cloud.githubusercontent.com/assets/13069084/22331242/9d51aaaa-e3f4-11e6-802b-e2e69b8b5036.png)

Запуск из командной строки

![shell](https://cloud.githubusercontent.com/assets/13069084/22331263/b79cdae2-e3f4-11e6-99ff-6897da574947.png)

```

**Comment by git22ura** [comment]
Предположу, что кириллица в пути к тестам

**Comment by pumbaEO** [comment]
нет, кривое арх решение с пробелами и кавычками в параметрах команды /C

**Comment by Veres72** [comment]
git22ura, как првоерить наверняка? В принципе все другие приложение, в т.ч. разные конфигурации, хранилище конфы и прочее работает на linux, с прилицей и без нареканий. 

pumbaEO, как быть?

**Comment by Veres72** [comment]
В инструкции указано, что параметр xddShutdown - не обязательный. Однако под linux без него не запускается. С ним - все ок. Для корректной работы под линукс команда должна выглядеть следующим образом: 
./1cv8 ENTERPRISE /IBConnectionString "Srvr=srv-name;Ref=db-name;" /NАдмин /P1234
/RunModeManagedApplication /Execute '/путь_к_ранеру/xddTestRunner.epf' /C 'xddRun ЗагрузчикКаталога /путь_к_тестам/xtest; 
xddReport ГенераторОтчетаJUnitXML /путь_к_отчету/report.xml; 
xddShutdown;'

Добавьте в инструкцию плиз.
